### PR TITLE
Feat/update doc

### PR DESCRIPTION
**Pull request by felixmusil**
I realized the how to add a representation has been broken by some of our recent changes and also it needed some additional clarifications (@max-veit). 

I would keep this branch alive and uptodate with master so that we can use it to work on the documentation when needed. Sounds good ?

### CODE REVIEW

File: bindings/bind_py_structure_manager.cc [hunk 1]
```diff
@@ -880,32 +880,35 @@ namespace rascal {
...
-    bind_make_structure_manager<Manager_t>(m_nl);
+    // list of already binded manager stacks to avoid double binding
     std::set<std::string> name_list{};
-    BindAdaptorStack<Manager_t, AdaptorNeighbourList, AdaptorStrict>
-        adaptor_stack_1{m_nl, m_adp, m_internal, name_list};
-
-    BindAdaptorStack<Manager_t, AdaptorNeighbourList, AdaptorCenterContribution,
+    /*-------------------- struc-man-bind-start --------------------*/
+    // bind the root structure manager
+    bind_structure_manager<StructureManagerCenters>(m_strc_mng, m_internal);
```
Comment (Luthaf) [inline bindings/bind_py_structure_manager.cc]: Why is the code updated too? Is this required for the doc changes ?

**Comment by Luthaf** [comment]
> I would keep this branch alive and uptodate with master so that we can use it to work on the documentation when needed. Sounds good ?

I am not sure what you mean by that. Could you expand on it?

File: bindings/bind_py_structure_manager.cc (hunk 1 above)
Comment (felixmusil) [reply bindings/bind_py_structure_manager.cc]: I reshuffled a bit the code so that it becomes more meaningful when displayed in the doc.

Comment (Luthaf) [reply bindings/bind_py_structure_manager.cc]: :+1:

**Comment by felixmusil** [comment]
> I am not sure what you mean by that. Could you expand on it?

I would not delete this branch and keep it around for when one feels like updating the doc.

**Comment by Luthaf** [comment]
What is the advantage of keeping the branch around versus creating a new one from master when one wants to change the docs? I can see at least one inconvenient with keeping a long-lived branch with a multiplication of merge commits.

**Pull request merged**